src/main/java/com/cst438/controller/EnrollmentController.java

```diff
@@ -65,6 +67,7 @@ public List<EnrollmentDTO> getEnrollments(
     // instructor uploads enrollments with the final grades for the section
     // user must be instructor for the section
     @PutMapping("/enrollments")
+    @PreAuthorize("hasAuthority('SCOPE_ROLE_INSTRUCTOR')")
     public void updateEnrollmentGrade(@RequestBody List<EnrollmentDTO> dlist) {
     
```

Comment:
I think we should pass in the Principal argument here so we can check that the caller of the method is actually the instructor of the section. See ```getAssignments``` method in assignmentController for a similar example.